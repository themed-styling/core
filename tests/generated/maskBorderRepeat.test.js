import 'jest-styled-components'

import { maskBorderRepeat } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(maskBorderRepeat, 'maskBorderRepeat')
