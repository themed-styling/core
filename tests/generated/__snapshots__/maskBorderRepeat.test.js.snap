// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maskBorderRepeat renders 0 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: 0;
  mask-border-repeat: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 0;
  mask-border-repeat: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 0;
  mask-border-repeat: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 0;
  mask-border-repeat: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders function 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: hasnot;
  mask-border-repeat: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: has;
  mask-border-repeat: has;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: hasnot;
  mask-border-repeat: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: has;
  mask-border-repeat: has;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders invalid theme path 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: invalid.path.1;
  mask-border-repeat: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: invalid.path.1;
  mask-border-repeat: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: invalid.path.1;
  mask-border-repeat: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: invalid.path.1;
  mask-border-repeat: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders negative number 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: -54881;
  mask-border-repeat: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: -54881;
  mask-border-repeat: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: -42365;
  mask-border-repeat: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: -42365;
  mask-border-repeat: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: July;
    mask-border-repeat: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: September;
    mask-border-repeat: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: July;
    mask-border-repeat: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: September;
    mask-border-repeat: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: May;
    mask-border-repeat: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: June;
    mask-border-repeat: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: April;
    mask-border-repeat: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: May;
    mask-border-repeat: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: June;
    mask-border-repeat: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-mask-border-repeat: April;
    mask-border-repeat: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    -webkit-mask-border-repeat: July;
    mask-border-repeat: July;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media January {
  .c0 {
    -webkit-mask-border-repeat: September;
    mask-border-repeat: September;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

@media June {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    -webkit-mask-border-repeat: July;
    mask-border-repeat: July;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media January {
  .c0 {
    -webkit-mask-border-repeat: September;
    mask-border-repeat: September;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: November;
    mask-border-repeat: November;
  }
}

@media June {
  .c0 {
    -webkit-mask-border-repeat: August;
    mask-border-repeat: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-mask-border-repeat: January;
    mask-border-repeat: January;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: December;
    mask-border-repeat: December;
  }
}

@media January {
  .c0 {
    -webkit-mask-border-repeat: April;
    mask-border-repeat: April;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: May;
    mask-border-repeat: May;
  }
}

@media June {
  .c0 {
    -webkit-mask-border-repeat: June;
    mask-border-repeat: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-mask-border-repeat: January;
    mask-border-repeat: January;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: December;
    mask-border-repeat: December;
  }
}

@media January {
  .c0 {
    -webkit-mask-border-repeat: April;
    mask-border-repeat: April;
  }
}

@media April {
  .c0 {
    -webkit-mask-border-repeat: May;
    mask-border-repeat: May;
  }
}

@media June {
  .c0 {
    -webkit-mask-border-repeat: June;
    mask-border-repeat: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders postitive number 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: 54881;
  mask-border-repeat: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 54881;
  mask-border-repeat: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 42365;
  mask-border-repeat: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: 42365;
  mask-border-repeat: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders string 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: July;
  mask-border-repeat: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: July;
  mask-border-repeat: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: June;
  mask-border-repeat: June;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: June;
  mask-border-repeat: June;
}

<div
    className="c0"
  />,
]
`;

exports[`maskBorderRepeat renders valid theme path 1`] = `
Array [
  .c0 {
  -webkit-mask-border-repeat: DarkSlateBlue;
  mask-border-repeat: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: DarkSlateBlue;
  mask-border-repeat: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: DarkSlateBlue;
  mask-border-repeat: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-mask-border-repeat: DarkSlateBlue;
  mask-border-repeat: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
